roles/nextflow/defaults/main.yml

```diff
@@ -1,13 +1,14 @@
-java_home: "/sw/comp/java/x86_64/sun_jdk1.8.0_151"
+java_home: /sw/comp/java/x86_64/sun_jdk1.8.0_151
 nextflow_java: "{{ java_home }}"
-nextflow_version_tag: "v21.04.3"
-nextflow_download_url: "https://github.com/nextflow-io/nextflow/releases/download/{{ nextflow_version_tag }}/nextflow"
+nextflow_version_tag: v21.04.3
+nextflow_download_url: https://github.com/nextflow-io/nextflow/releases/download/{{ nextflow_version_tag }}/nextflow
 nextflow_env:
-  NXF_LAUNCHER: '$PWD/work'
-  NXF_LAUNCHBASE: '$PWD/work'
-  NXF_HOME: "{{ nextflow_dest }}/workfiles"
-  NXF_WORK: /scratch
-  NXF_TEMP: /scratch
+  NXF_LAUNCHER: /scratch/{{ deployment_version }}
```

Comment:
You don't miss much 😄 

I don't think this matters too much. I added this path element because I actually don't think that the scratch area is cleared very often (perhaps not even after a reboot?) and I just wanted to make extra sure that there wouldn't be unexpected effects because of cross-pollination between deployments. On miarka3 in particular, I'm doing quite a lot of test deployments and whatnot right now.

Reply:
:+1: